.travis.yml

```diff
@@ -161,7 +155,7 @@ before_script:
   - if [ "$DB" = 'pgsql' ]; then psql -c 'CREATE DATABASE bedita_test;' -U postgres; fi
   - if [ "$DB" = 'pgsql' ] && [ "$DB_POSTGIS" = '1' ]; then psql -c 'CREATE EXTENSION postgis;' -U postgres; fi
 
-script: vendor/bin/phpunit --coverage-clover=clover.xml # Run PHPUnit.
```

Comment:
a new `reference` stage is introduced with PHP 7.1, MySQL 5.7 and code coverage with codecov upload - all other builds will not have coverage